modules/core/src/main/resources/default.scala-steward.conf

```diff
@@ -134,5 +134,8 @@ updates.ignore = [
   { groupId = "org.http4s", version = "0.19.0" },
 
   // https://github.com/scala-js/scala-js/issues/3865
-  { groupId = "org.scala-js", version = "0.6.30" }
+  { groupId = "org.scala-js", version = "0.6.30" },
+  
+  // https://github.com/scala-steward-org/scala-steward/issues/2708
+  { groupId = "com.typesafe.akka", version = { prefix = "2.7." } }
```

Comment:
FYI: @ckipp01 ignored two more groupIds in https://github.com/akka/akka/pull/31561#issuecomment-1239052679. Maybe it is safer to add them too.

Reply:
Btw, if Akka is ignored like the above, Scala Steward will prevent updates to 2.7 but will propose updates to 2.8 or later when they are available.

Reply:
Looks like the other ones have a lot of different versions :thinking: so we would need to ignore them per artifact or ignore them altogether for now and work on license checking. What do you think?

Reply:
> Btw, if Akka is ignored like above, Scala Steward will prevent updates to 2.7 but will propose updates to 2.8 or later when they are available.

ach, that's right. So maybe ignore all for now?

Reply:
I changed it to all the groupIds I found. I will try to work on the licenses next week.

Reply:
Instead of ignoring, why not pinning to `2.6`? This is what we do in Typelevel for Scala 3 updates.
- https://github.com/typelevel/steward/pull/10

Reply:
Wasn't sure what pinning would entail. Would it only update it up until the said version?

Also I would need to go through all the libraries since they have different versions where they start being BSL :/

Probably best approach would be to ignore it for now and then add the configuration for licenses.

Reply:
@fthomas what do you think? Is it ok to ignore for now and fix the license later? Alternatively, we can go trough the akka libs one by one and pin all of them.

We could also only change the config for the public instance of Scala Steward, however, we would need to make sure in that case that any changes to the default config are upstreamed.

Reply:
> @fthomas what do you think? Is it ok to ignore for now and fix the license later?

I'm OK with that.

Reply:
Awesome! I will make sure that this is only temporary and work on the license as soon as possible.